Docs: Meetup Guidelines (#13520)
apache/iceberg: site/docs/community.md

[open-issues]: https://github.com/apache/iceberg/issues
[new-issue]: https://github.com/apache/iceberg/issues/new

Looking to contribute to Apache Iceberg? See [Contributing](contribute.md) for more details on how to get started.

## Connect with Community Events 

The Apache Iceberg community regularly meets for technical discussion both in-person and virtually.

### Hosting an Apache Iceberg Meetup

Meetups related to the Apache Iceberg project are held around the globe thanks to community volunteers. Interested individuals are allowed to host a community meetup where the name **Apache Iceberg Meetup <geographical location\>** may be used in
compliance with [Apache Software Foundation's branding and trademarks guidelines](https://www.apache.org/foundation/marks/events.html#approval). No explicit PMC approval is required.

> In addition, hosts are required to ensure that their event abides by the following: 

Reviewer: Not sure we need "In addition," here

```suggestion
Hosts are required to ensure that: 
```

1. The Apache Iceberg ecosystem should be championed in every meetup and technical session 
2. All talks should be vendor-neutral and not sales pitches
3. Each meetup should have at least two talks with speakers representing different companies/organizations
4. Planned meetups ought to be brought to the attention of the [dev list](#mailing-lists)

> 5. All [Community Guidelines](#community-guidelines) must be respected

Reviewer 2: +1, thanks for adding this

### Apache Iceberg Community Calendar

> This calendar contains two calendar feeds:

Reviewer: ```suggestion
The community maintains two calendar feeds:
```

?

> * [Iceberg Dev Events](https://calendar.google.com/calendar/u/0?cid=MzkwNWQ0OTJmMWI0NTBiYTA3MTJmMmFlNmFmYTc2ZWI3NTdmMTNkODUyMjBjYzAzYWE0NTI3ODg1YWRjNTYyOUBncm91cC5jYWxlbmRhci5nb29nbGUuY29t){:target="_blank"}: Events such as the triweekly Iceberg sync, aimed to discuss the project roadmap and how to implement features.

Reviewer 2: nit:
```suggestion
* [Iceberg Dev Events](https://calendar.google.com/calendar/u/0?cid=MzkwNWQ0OTJmMWI0NTBiYTA3MTJmMmFlNmFmYTc2ZWI3NTdmMTNkODUyMjBjYzAzYWE0NTI3ODg1YWRjNTYyOUBncm91cC5jYWxlbmRhci5nb29nbGUuY29t){:target="_blank"}: A shared calendar for all Iceberg development syncs, including the triweekly community sync, subproject syncs for Python, Rust, and Go, the Iceberg Catalog Community Sync, and other ad hoc syncs the community may want to schedule or discuss.
```

we added all dev related events to this calendar 
context: https://lists.apache.org/thread/fnj2wjk7wwbonc82znkx7r9xs9mfkddf

Author: Updated!

> * [Iceberg Community Events](https://calendar.google.com/calendar/u/0?cid=NTkzYmIwMGJmZTQ1N2QzMTkxNDEzNTBkZDI0Yzk2NGYzOWJkYmQ5ZmQyNDMyODFhODYzMmEwMDk2M2EyMWQ4NkBncm91cC5jYWxlbmRhci5nb29nbGUuY29t){:target="_blank"}: Events such as conferences and meetups, aimed to educate and inspire Iceberg users.

Reviewer 3: Is it possible to add a link to https://lu.ma/apache-iceberg or each geographical community pages so readers can find new events? I know the Luma site is  "unofficial", though. 

(No need to handle in this PR)

Reviewer 2: +1 i think we should just use the "Iceberg Community Events". maybe theres a way to sync from the luma calendar to this

Hosts are required to ensure that: 

1. The Apache Iceberg ecosystem should be championed in every meetup and technical session 
2. All talks should be vendor-neutral and not sales pitches
3. Each meetup should have at least two talks with speakers representing different companies/organizations
4. Planned meetups ought to be brought to the attention of the [dev list](#mailing-lists)
5. All [Community Guidelines](#community-guidelines) must be respected

### Apache Iceberg Community Calendar

The community maintains two calendar feeds:

* [Iceberg Dev Events](https://calendar.google.com/calendar/u/0?cid=MzkwNWQ0OTJmMWI0NTBiYTA3MTJmMmFlNmFmYTc2ZWI3NTdmMTNkODUyMjBjYzAzYWE0NTI3ODg1YWRjNTYyOUBncm91cC5jYWxlbmRhci5nb29nbGUuY29t){:target="_blank"}: Events such as the triweekly Iceberg sync, aimed to discuss the project roadmap and how to implement features.
* [Iceberg Community Events](https://calendar.google.com/calendar/u/0?cid=NTkzYmIwMGJmZTQ1N2QzMTkxNDEzNTBkZDI0Yzk2NGYzOWJkYmQ5ZmQyNDMyODFhODYzMmEwMDk2M2EyMWQ4NkBncm91cC5jYWxlbmRhci5nb29nbGUuY29t){:target="_blank"}: Events such as conferences and meetups, aimed to educate and inspire Iceberg users.

> Additional community-run meetups can be found on the [unofficial Apache Iceberg™ and Ecosystem Events Luma calendar](https://lu.ma/apache-iceberg).

Reviewer: I'm not sure about this one, just because it's not official.
